build.gradle

```diff
@@ -129,10 +128,7 @@ dependencies {
     //required by lsp4j as the version from IJ is incompatible
     compile 'com.google.code.gson:gson:2.8.9'
     compile 'com.vladsch.flexmark:flexmark:0.50.50'
-    integrationTestImplementation("com.intellij.remoterobot:remote-robot:$remoteRobotVersion")
-    integrationTestImplementation("com.intellij.remoterobot:remote-fixtures:$fixturesVersion")
-    integrationTestImplementation 'org.junit.jupiter:junit-jupiter-api:5.3.1'
-    integrationTestRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.3.1'
+    compile 'com.redhat.devtools.intellij:intellij-common-ui-test-library:0.0.8'
```

Comment:
If it's in compile scope then it's going to finish in the plugin ZIP file ?

Reply:
Yes, it will, I have tested that on localhost. Do you have an idea/suggestion on improving that? Another way that works for UI tests+lib connection but does not include that in the plugin ZIp file?

Reply:
I see the **testCompile** which could probably work. After `./gradlew buildPlugin` the library is not in the plugin ZIP file. I have never used that, do you think I should replace that?

Reply:
Yes I see no reason to have that lib in the production ZIP file